cratedb_toolkit/admin/xmover/analysis/shard.py

```
@@ -836,6 +837,72 @@
+        shards: list[ShardInfo] = self.analyzer.shards
+
+        # Cluster summary table
+        shards_table = Table(title="Hot shards", box=box.ROUNDED)
+        shards_table.add_column("Schema", style="cyan")
+        shards_table.add_column("Table", style="cyan")
+        shards_table.add_column("ID", style="cyan")
+        shards_table.add_column("Node", style="cyan")
+        shards_table.add_column("Primary", style="cyan")
+        shards_table.add_column("Size", style="magenta")
+        shards_table.add_column("Max Seq_no", style="magenta")
+        # shards_table.add_column("Global Checkpoint", style="magenta")
```

Ruff (0.12.2)

Found commented-out code

Remove commented-out code

(ERA001)